Userland/Libraries/LibWeb/HTML/HTMLInputElement.cpp

```diff
@@ -1580,7 +1574,9 @@ unsigned HTMLInputElement::size() const
 
 WebIDL::ExceptionOr<void> HTMLInputElement::set_size(unsigned value)
 {
-    return set_attribute(HTML::AttributeNames::size, MUST(String::number(value)));
+    auto result = set_attribute(HTML::AttributeNames::size, MUST(String::number(value)));
+    document().invalidate_layout();
+    return result;
 }
```

Comment:
Hmm, this seems like it would trigger on
```js
element.size = 3;
```
but not on
```js
element.setAttribute("size", 3);
```

Reply:
Oh darn great point. Do you think we should just add a special case for `size` in `set_attribute?`